### DISCUSSION TITLE
Welcome to Dev Lake Discussions!

### DISCUSSION THREAD

**Discussion by basicthinker** [apache/devlake#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Urvesh0890** [comment]
What are the type of databases that can be used to store data in devlakes?

**Reply by klesh** [reply]
Currently, we are supporting mysql8 and postgres

**Reply by vorakumar** [reply]
Where do I find example configurations to use postgres as a data store?

**Reply by ZhangNing10** [reply]
in the helm chart, postgres part has been commented, we will soon release a helm chart with postgresql options

**Reply by ZhangNing10** [reply]
hello @vorakumar I am sorry, helm chart would not support postgresql options according to https://github.com/apache/incubator-devlake-helm-chart/pull/240, https://github.com/apache/incubator-devlake-helm-chart/issues/148

**Reply by soumya-angelone** [reply]
Does Devlake capture git release version tags as we want to showcase rollbacks.

**Reply by Startrekzky** [reply]
@soumya-angelone Not yet. You can put up a feature request with your concrete use case.
